apartments/tests.py

```diff
@@ -61,6 +61,11 @@ def test_get_apartment(self, new_apartment):
         check_apartment = Apartment.objects.get(owner=new_apartment.owner)
         assert check_apartment == new_apartment
 
+    def test_get_all_relevant_apartments(self):
+        apartments_qurey = Apartment.get_all_relevant_apartments()
+        assert all(isinstance(current_apartment, Apartment) for current_apartment in apartments_qurey)
```

Comment:
I don't see any insertion of test data into the database.
what would happen if there no apartments in it?

Reply:
I thought the test data was used for that.. but I understand from you that this is not happening.
So I will add insertion of test data.

Reply:
Please double-check me on that.

Reply:
Generally, we accept that the test data inserted into the system is expected to be there from the tests perspective.

Reply:
ack, I'm resolving this, but fix the conflicts so we'll be able to merge @tamirho